glide.yaml

```diff
@@ -2,23 +2,28 @@ package: github.com/kubeless/kubeless
 import:
 - package: github.com/sirupsen/logrus
 - package: github.com/spf13/cobra
+  version: v0.0.1
 - package: k8s.io/apimachinery
-  version: 9d38e20d609d27e00d4ec18f7b9db67105a2bde0
+  version: kubernetes-1.8.7
 - package: k8s.io/client-go
+  version: kubernetes-1.8.7
```

Comment:
hey I think you made a mistake here. We pinned client-go to v5.0.0. Now there are 2 versions declared for this package.

Reply:
@ngtuna My bad! Let me create a quick PR.